trunk/NDHMS/Routing/module_channel_routing.F

```diff
@@ -94,7 +94,7 @@ subroutine LEVELPOOL(ln,qi0,qi1,qo1,ql,dt,H,ar,we,maxh,wc,wl,oe,oc,oa)
 real :: dh, dh1, dh2, dh3      ! Depth in weir, and height function for 3 order RK 
 real :: It, Itdt_3, Itdt_2_3   ! inflow hydrographs 
 real :: maxWeirDepth           !maximum capacity of weir
-real :: qldum                  !dummy ql to 0 out
+!real :: hdiff_vol, qdiff_vol   ! water balance check variables
```

Comment:
If adding new variables to the code, can we make an effort to move away from F77 style/conventions? Pick a descriptive name and add a descriptive inline comment with units.  Is qdiff_vol a flow? A Rate? A Volume?  Similiarly, hdiff_vol is not informative, I can't determine if it is a height, head, or volume.  I recommend adding them each on their own line with elaborated names and inline comments.

Reply:
Sure, although these were really just our internal testing checks that I left in just in case we need to activate again. This is not really how we should be doing mass balance checks on this routine, which is why it is commented out. Might be cleaner just to remove completely - this is really just a quick patch to get these updates into ops since I know you all are working on a full refactor for next round...